src/lib/features/event-generator/stores/EventGeneratorSteps.ts

```diff
@@ -59,6 +60,17 @@ export const eventGeneratorSteps = createSteps([
 		title: 'Give super powers to your FLOAT',
 		action: null,
 		state: 'inactive',
+		button: {
+			text: 'Review Event',
+			icon: 'tabler:arrow-right'
+		}
+	},
+	{
+		name: 'Review',
+		component: ReviewStep,
+		title: 'Review your event prior to confirming',
+		action: null,
```

Comment:
Here is where the action should be placed.
It'll then be called automatically when we change the step.

If this causes any problem with the implementation, we can leave it the way you did it.
Maybe, if we place the action here when it is called, the FLOAT we want to screenshot has already gone away from the screen, so we can't do it this way..